src/freenet/client/filter/ElementInfo.java

```diff
@@ -218,7 +227,27 @@ public class ElementInfo {
 			"first-letter",
 			"before",
 			"after",
-			"target"
+			"target",
```

Comment:
target looks dangerous (it checks which link is used to access a page). It can be used to trigger background fetches when navigating (just verified: the network request is sent when the target is clicked), so it may be necessary to restrict its content to non-links to avoid timing attacks.

Reply:
target is usually used to highlight a section heading. Clicking different links should already trigger different network effects because they can link to different pages, yeah.

Reply:
You’re right — if they get the link from outside, it could just be different links, and if they click it on the page, it could lead anywhere in the net.

⇒ good to go. Thank you!